core/modules/field/modules/list/list.module

```diff
@@ -343,8 +343,29 @@ function list_extract_allowed_values($string, $field_type, $generate_keys) {
  */
 function list_allowed_values_string($values) {
   $lines = array();
+  $previous_key = false;
```

Comment:
Needed to fix error reported in https://www.drupal.org/project/options_element/issues/2454469 when there are optgroups. It borrows from `_form_options_to_text()` but doesn't have $key_type so can't just reuse it.

Reply:
Note this should be `FALSE` (uppercase)